addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -970,7 +970,7 @@ export function isUnremovable(node) {
     return (
         isEditableRoot ||
         (node.nodeType === Node.ELEMENT_NODE &&
-            (node.getAttribute('t-set') || node.getAttribute('t-call'))) ||
+            (node.classList.contains('o_editable') || node.getAttribute('t-set') || node.getAttribute('t-call'))) ||
```

Comment:
> You know better than me but so that I understand: I see no mention of the "o_editable" class in the "lib" repo of the editor, which makes me think it is a class of Odoo (web_editor)... should this file even know about it?

In theory, this file should not know about it. In practice, the lib already hardcode Odoo specifics.
I usually try as much a possible to avoid it but in this case, parametrizing it would require too much changes.